src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -578,6 +602,21 @@ public FlagUsageException(String message) {
    * places.
    *
    * @param files A list of filenames.
+   * @param jsonFiles A list of json encoded files.
```

Comment:
Fix the first line of this comment to not mention zips (github won't let me add this on that line).